perception/traffic_light_map_based_detector/src/node.cpp

```diff
@@ -1,4 +1,4 @@
-// Copyright 2020 Tier IV, Inc.
+// Copyright 2023 Tier IV, Inc.
```

Comment:
```suggestion
// Copyright 2023 TIER IV, Inc.
```

Reply:
updated in commit 2606f617ff638fa0cd866f0f0c058e464ee049a9